### ISSUE TITLE
[OneWire] Missing support for `ignore85CPowerOnResetValues`

### ISSUE THREAD

**Issue by mgwoj** [eclipse/smarthome#6344]
Version 1 of OneWire binding has option `ignore85CPowerOnResetValues` to ignore invalid readings from DS18x20 sensor. Such invalid reading is visible as temperature of 85oC.

**Comment by J-N-K** [comment]
This is only for the DS18x20 sensors, right?

**Comment by mgwoj** [comment]
No, I also have temperature readings of +85°C for DS1923, even if there is no such information in the datasheet [DS1923.pdf](https://datasheets.maximintegrated.com/en/ds/DS1923.pdf)

**Comment by J-N-K** [comment]
Ok. So we could add that to the temperature channel as option. Sounds easy. I‘ll have a look  in the next days.
